### ISSUE TITLE
[Bug] Responsive Desgin Bugs

### ISSUE THREAD

**Issue by sumitbishti** [Sidd2024/Allops#2]
Hand Pointer should occur when hovering over the links in socials section.
![Screenshot (30)](https://user-images.githubusercontent.com/75713174/193198188-442e6f94-b5f7-4ca7-836f-d57f80cb98cf.png)

Sign in/Sing up buttons not aligned properly on smaller screens.
![Screenshot (29)](https://user-images.githubusercontent.com/75713174/193198200-3dcc6ea9-f965-4bfa-8754-e960cbb5c083.png)

I can fix these issues.

**Comment by Sidd2024** [comment]
I haven't added the links for twitter and youtube

**Comment by Sidd2024** [comment]
Yeah there are a lot of issues with mobile responsiveness so please fix them..

**Comment by sumitbishti** [comment]
> I haven't added the links for twitter and youtube

Yes, but you haven't even created the boilerplate for them. Never mind, i will fix them. Just assign appropriate labels onto them.

**Comment by Pranshu1902** [comment]
I would like to work on this issue under hacktoberfest, please assign it to me

**Comment by Ujjawal-Kothari** [comment]
Hey!

**Comment by Ujjawal-Kothari** [comment]
Hey! I want to work on this issue under Hacktoberfest please assign it to me.

**Comment by Pranshu1902** [comment]
@Sidd2024 I have made a PR #24  for this issue, please look into it and verify it.
